src/utils/process.js

```diff
@@ -65,30 +64,41 @@ async function findProcessUnix (browserId) {
     return findProcessIdUnix(browserId, output);
 }
 
-async function checkUnixProcessIsKilled (processId) {
+async function isUnixProcessKilled (processId) {
     const output = await getProcessOutputUnix();
 
-    if (isProcessExistUnix(processId, output)) {
-        await delay(CHECK_KILLED_DELAY);
+    return !isUnixProcessExist(processId, output);
+}
 
-        await checkUnixProcessIsKilled();
-    }
+async function killUnixProcessSoft (processId) {
+    process.kill(processId);
+}
+
+async function killUnixProcessHard (processId) {
+    process.kill(processId, 'SIGKILL');
 }
 
 async function killProcessUnix (processId) {
-    let timeoutError = false;
+    let softTries = 0;
+    let unixProcessKilled = false;
 
-    process.kill(processId);
+    do {
+        await killUnixProcessSoft(processId);
+        softTries++;
+        await delay(CHECK_KILLED_DELAY);
+        unixProcessKilled = await isUnixProcessKilled(processId);
+    } while (!unixProcessKilled && softTries < 2);
 
-    const killTimeoutTimer = delay(CHECK_PROCESS_IS_KILLED_TIMEOUT)
-        .then(() => {
-            timeoutError = true;
-        });
+    unixProcessKilled = await isUnixProcessKilled(processId);
+    if (unixProcessKilled) return;
```

Comment:
Please, add more whitespace to make your code consistent with the all other code and improve readability:
```suggestion
    let softTries         = 0;
    let unixProcessKilled = false;

    do {
        await killUnixProcessSoft(processId);
        
        softTries++;
        
        await delay(CHECK_KILLED_DELAY);
        
        unixProcessKilled = await isUnixProcessKilled(processId);
    } 
    while (!unixProcessKilled && softTries < 2);

    unixProcessKilled = await isUnixProcessKilled(processId);
    
    if (unixProcessKilled)
        return;
```